client/src/components/PortfolioTable.tsx

```diff
@@ -587,7 +606,7 @@ export const PortfolioTable = React.memo((props: PortfolioTableProps) => {
             ))}
           </thead>
           <tbody>
-            {table.getRowModel().rows.length &&
+            {table.getRowModel().rows.length ? (
```

Comment:
does this get fixed if you `parseInt(table.getRowModel().rows.length)`? 
it's weird that the 0 is being logged as a valid length and i wonder if it's a underlying type issue

Reply:
yeah this was the issue - when filters are applied that exclude all properties the length is `0` and so that's falsey in the `&&` and so the `0` goes through here. I'm not sure I know what you mean about the type issue - like it's weird that the list of rows can be empty?

Reply:
ah ok, thanks for clarifying. I don't think that there have been issues with the type from `rows`, so this `!!` should deal with it all and make it more clear.